packages/js/product-editor/src/components/tabs/tabs.tsx

```diff
@@ -31,13 +27,6 @@ export function Tabs( { onChange = () => {} }: TabsProps ) {
 	const [ selected, setSelected ] = useState< string | null >( null );
 	const query = getQuery() as Record< string, string >;
 
-	function onClick( tabId: string ) {
-		window.document.documentElement.scrollTop = 0;
```

Comment:
Any reason we removed the scroll to top on navigation?  I'm no longer being scrolled to the top when navigating to a new tab.

Reply:
I removed this line since I noticed that the `navigateTo()` utility actually includes that line as well, so it was redundant. On looking at this further I'm realizing that it wasn't working to begin with. This is due to the fact that the form scrolling is actually from the `overflow: auto` on the interface-skeleton body, and not the document. 

This is actually all seeming a bit odd, as the way it's setup the global sidebar is actually not fixed, but the interface skeleton container is. It's then creating a scrollable body container where our form lives. This _could_ be changed to make the sidebar fixed (as it is with the previous react and legacy editors), but that would be diverging from the blocks way, I suppose. It's also creating the dual scrollbars on the right-hand side of the screen:

![image](https://user-images.githubusercontent.com/444632/236041778-48379efa-0d27-4090-ba3d-91b0444260a5.png)

We may want to decide first if we want to considering changing anything on a larger scale here to address both of those issues, or work around them. We can't simply use a `getScrollContainer()` from `@wordpress/dom` here since it's in a slot-fill, but there are other ways to work around it of course. 

Perhaps a follow-up then?